api/interfaces/init.go

```diff
@@ -11,3 +11,16 @@ type Zone struct {
 	Number         int                `json:"number" bson:"number"`
 	Gym            primitive.ObjectID `json:"gym" bson:"gym"`
 }
+
+type SignUpForm struct {
+	User     string `json:"user"`
+	Email    string `json:"email"`
+	Password string `json:"password"`
+}
+type User struct {
```

Comment:
Can you add the isVerified field? The default value should be false, @SilviaPabon will take care of the email verification endpoint.